_posts/2024-10-28-boosting-cde-security-on-k8s-cluster.adoc

```diff
@@ -0,0 +1,80 @@
+---
+title: "Boosting CDE Security on k8s cluster: A Guide to new features"
+layout: post
+author: Anatolii Bazko
+description: >-
+  Boosting CDE Security on k8s cluster: A Guide to new features
+categories: []
+keywords: ['workspace', 'security', 'administrator']
+slug: /@tolusha/boosting-cde-security-on-k8s-cluster
+---
+
+== Introduction
+
+As organizations transition to cloud-native development, maintaining control, security, and flexibility in development environments becomes critical. Eclipse Che is a powerful Kubernetes-native IDE that offers administrators tools for ensuring security and resource efficiency. In this post, we'll look at three new Eclipse Che features that improve security and resource control in your development environments.
```

Comment:
```suggestion
As organizations transition to cloud-native development, maintaining control, security, and flexibility in development environments becomes critical. Eclipse Che is a powerful Kubernetes-native IDE that offers administrators tools for ensuring security and resource efficiency. In this post, we'll look at three new Eclipse Che features that improve security and resource control in your development environments.
```

Blog post presents three new features and I believe we need three dedicated section e.g. "Untrusted Repository Warning" could be added explicitly